addon/components/place-autocomplete-field.js

```diff
@@ -162,6 +166,12 @@ export default Component.extend({
     }
   },
 
+  _bindDataAttributesToInput() {
+    let properties = A(Object.keys(this)).filter((prop) => prop.indexOf('data-') >= 0) || [];
```

Comment:
Why do we need to wrap this on `A()`?

Reply:
Ups, we don't need it. I forgot to remove that.